libs/api/domains/education/src/lib/graphql/frigg/option.model.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+import { FriggValueModel } from './value.model'
+
+@ObjectType()
+export class FriggOptionModel {
+  @Field(() => String)
+  id!: string
+
+  @Field(() => String)
+  key!: Array<FriggOptionModel>
```

Comment:
see above (also, if it was required it should've been `=> [String]`)

Reply:
Fixed